multimodal/src/autogluon/multimodal/utils/export.py

```diff
@@ -130,13 +131,14 @@ def export_onnx(
         input_keys = self._model.input_keys
         input_vec = [batch[k] for k in input_keys]
 
-        # Infer default onnx path, and create parent directory if needed
+        # Write to BytesIO if path argument is not provided
         if not path:
```

Comment:
I would recommend changing this to `if path is None` explicitly to avoid bugs as python is not a strong typed language and many things could trigger the `if not path` clause.

Reply:
changed as suggested.